src/components/root/Navigation/SideBar.tsx

```
@@ -17,8 +18,14 @@ export interface SideBarProps {
 export default async function SideBar({ children, initialStoreProps }: { initialStoreProps?: SidebarStoreProviderProps['initialStoreProps']; children: ReactNode }) {
   return (
     <SidebarStoreProvider initialStoreProps={initialStoreProps}>
-      <DesktopSidebar children={children} className='hidden md:flex' />
-      <MobileSideBar children={children} visibilityBreakpoints='flex md:hidden opacity-100 md:opacity-0' />
+      <ConditionalBreakpointRendering hideBreakPoint='768px' logIdentifier='mobile-sidebar'>
+        <MobileSideBar children={children} />
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)